runtime/mozart/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+[package]
+name = "tanssi-relay"
+default-run = "tanssi-relay"
+description = "Implementation of a `https://tanssi.network` node in Rust based on the Substrate framework."
+edition = "2021"
+license = "GPL-3.0-only"
+readme = "README.md"
+version = "0.1.0"
+
+[[bin]]
+name = "tanssi-relay"
+path = "src/main.rs"
+
+[lints]
+workspace = true
+
+[dependencies]
+color-eyre = { workspace = true, default-features = false }
+tikv-jemallocator = { workspace = true, optional = true, features = [ "unprefixed_malloc_on_supported_platforms" ] }
+
+# Crates in our workspace, defined as dependencies so we can pass them feature flags.
+polkadot-node-core-pvf = { workspace = true }
+polkadot-node-core-pvf-prepare-worker = { workspace = true }
+polkadot-overseer = { workspace = true }
+tanssi-relay-cli = { workspace = true, features = [ "mozart-native" ] }
+
+# Needed for worker binaries.
+polkadot-node-core-pvf-common = { workspace = true }
+polkadot-node-core-pvf-execute-worker = { workspace = true }
```

Comment:
@tmpolaczyk I can also remove these and the `src/bin/*-worker.rs` files. For now I need them due to architecture issues but will remove before merging.

Reply:
If you have the compiled binaries you should be able to remove those files, just modify the download-polkadot.sh script to skip downloading if the architecture is arm and it will keep working locally for you.